lib/api/core/models/repositories/tokenRepository.js

```diff
@@ -248,6 +248,18 @@ class TokenRepository extends Repository {
         }));
       });
   }
+
+  /**
+   * The repository main class refreshes automatically the TTL
+   * of accessed entries, letting only unaccessed entries expire
+   *
+   * But tokens' TTL must remain the same than their expiration time,
+   * refreshing a token entry has no meaning.
+   *
+   * So we need to override the TTL auto-refresh function to disable it
+   */
+  refreshCacheTTL() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this function is empty or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1186)